src/main/java/de/muenchen/isi/infrastructure/entity/Baurate.java

```diff
@@ -27,6 +28,6 @@ public class Baurate extends BaseEntity {
     @Column(precision = 10, scale = 2, nullable = true)
     private BigDecimal geschossflaecheWohnenGeplant;
 
-    @Embedded
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
`CascadeType.ALL`:

Wird die Baurate gelöscht wird somit auch der Fördermix in der Fördermixtabelle gelöscht. Der Fördermixstamm verliert somit seine referenzierte Entität.

Im umgekehrten Fall beim Löschen eines Fördermixstamms verliert die Baurate ebenso seine referenzierte Entität.

Reply:
Vermutlich ist es besser wenn der Fördermix mit den darin enthaltenen Förderarten als @Embedabble behandelt wird.

In der GUI für den User (nicht Admin) wird dann der gewünschte FördermixStamm ausgewählt. Bei der Auswahl wird das Attribut Fördermix einfach kopiert und der Baurate zugewiesen. Somit existiert die Baurate mit einer Kopie eines Fördermixes (ausgewählt aus einem Fördermixstamm). Folglich besteht nicht mehr die Gefahr, eines CascadeDelete und somit eines Verlusts einer Referenz.